src/main.ts

```diff
@@ -482,6 +482,26 @@ export async function readPipeline(config: Config): Promise<string> {
   const filename = path.join(folderName, config.pipeline)
   core.debug(`Reading pipeline file from ${filename}`)
   let pipeline = fs.readFileSync(filename).toString()
+  const pipelineJson = JSON.parse(pipeline)
+  const runtimeParametersFile = pipelineJson.phases.verify?.context.runtime_parameters_file
+  if (runtimeParametersFile !== undefined) {
+    const runtimeParametersFilePath = path.join(folderName, runtimeParametersFile)
+    let runtimeParameters = Buffer.from(fs.readFileSync(runtimeParametersFilePath).toString().trim()).toString("base64")
```

Comment:
Another interesting one, we probably want to run variable substitution to the contents of the file before encoding it to base64. That way users will be able to inject secrets on the same template variables that we use on pipelines.